net/nginx-util/src/nginx-ssl-util.cpp

```diff
@@ -0,0 +1,515 @@
+#include <thread>
+#ifdef NO_PCRE
+#include <regex>
+#else
+#include "regex-pcre.hpp"
+#endif
+#include "px5g-openssl.hpp"
+#include "nginx-util.hpp"
+
+using namespace std;
+
+static const auto CRON_INTERVAL = string{"3 3 12 12 *"}; // once a year.
```

Comment:
This looks like "static const" can be simply replaced with constexpr on first look.

Reply:
Replaced them by static constexpr … std::string_view …